Powershell scripts/Defender for Servers on resource level/ResourceLevelPricingAtScale.ps1

```diff
@@ -159,15 +159,15 @@ if ($continue.ToLower() -eq "n") {
 }
 
 Write-Host "-------------------"
-$PricingTier = Read-Host "Enter the pricing tier to set these resources - 'Free' or 'Standard' (choosing 'Standard' will enable the machines with 'P1' subplan)"
-while($PricingTier.ToLower() -ne "free" -and $PricingTier.ToLower() -ne "standard"){
-$PricingTier = Read-Host "Enter the pricing tier to set these resources - 'Free' or 'Standard' (choosing 'Standard' will enable the machines with 'P1' subplan)"
+$PricingTier = Read-Host "Enter the command set these resources - 'Free' or 'Standard' or 'Delete' or 'Read' (choosing 'Free' will remove the Defender protection; 'Standard' will enable the 'P1' subplan; 'Delete' will remove any existing inheriting exception; 'Read' will read the current inheriting configuration)"
```

Comment:
'Delete' will remove any explicitly set configuration (the resource will inherit the parent's configuration); 'Read' will read the current configuration.

Reply:
@stefanpems please address above feedback so the PR can be approved.